src/renderers/dom/test/__tests__/ReactTestUtils-test.js

```diff
@@ -844,4 +844,20 @@ describe('ReactTestUtils', () => {
       );
     });
   });
+
+  it('should call setState with the correct parameters', () => {
+    let mockArg = 'the wrong value';
+    let mockCallback = arg => (mockArg = arg);
```

Comment:
so I tried this but `npm test` causes the length of arguments to be 1, `[ undefined ]`.  But `node ./scripts/fiber/record-tests` has the length of the arguments as 0, `[ ]`

So it seems like checking the first argument is the only way to go that I see